drivers/flash/Kconfig.mcux

```diff
@@ -11,3 +11,16 @@ config SOC_FLASH_MCUX
 	  the duration of the flash erase/write operations. This will
 	  have an impact on the overall system performance - whether
 	  this is acceptable or not will depend on the use case.
+
+if SOC_FLASH_MCUX
+
+config SW_MANAGED_FLASH_WRITE
+	bool "MCUX flash software managed write"
+	default y
```

Comment:
I'm very unwilling to allow this code in at all. It certainly should not be a "default y". It really shouldn't be present at all.

Reply:
@d3zd3z i accidentally force pushed this to 'y' with some generated file, will set back to 'n'